### PR TITLE
Migrate/Cleanup hlm

### PR DESCRIPTION
**Pull request by Flova**
## Necessary checks
- [ ] Update package version
- [ ] Run `catkin build`
- [ ] Write documentation
- [ ] Create issues for future work
- [ ] Test on your machine
- [ ] Test on the robot
- [ ] Put the PR on our Project board

### CODE REVIEW

File: msg/GameState.msg [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# This message provides all information from the game controller
+# for additional information see documentation of the game controller
+# https://github.com/bhuman/GameController
+
+
+std_msgs/Header header
+
+uint8 GAMESTATE_INITAL=0
```
Comment (jaagut) [inline msg/GameState.msg]: Please fix this typo (see https://github.com/bit-bots/humanoid_league_msgs/issues/37)

File: msg/RobotRelative.msg [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# An another robot relative to the robot
+
+# robot without known color
+uint8 ROBOT_UNDEFINED = 1
+uint8 ROBOT_MAGENTA = 2
+uint8 ROBOT_CYAN = 3
```
Comment (jaagut) [inline msg/RobotRelative.msg]: The rules (https://humanoid.robocup.org/wp-content/uploads/RC-HL-2023-Rules.pdf line 371pp) state that the colors are defined as `red` and `blue`. Please rename to the correct names.

**Review by jaagut — COMMENTED** [review]
Does this also address https://github.com/bit-bots/humanoid_league_msgs/issues/36 ?

**Review by jaagut — APPROVED** [review]

**Review by jaagut — APPROVED** [review]

File: msg/RobotRelative.msg (hunk 2 above)
Comment (jaagut) [reply msg/RobotRelative.msg]: Done.

File: msg/RobotRelative.msg [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 # robot without known color
 uint8 ROBOT_UNDEFINED = 1
-uint8 ROBOT_MAGENTA = 2
-uint8 ROBOT_CYAN = 3
+uint8 ROBOT_RED = 2
```
Comment (Flova) [inline msg/RobotRelative.msg:5]: Are these labels utilized elsewhere?

Comment (jaagut) [reply msg/RobotRelative.msg:5]: Yes, changed them too

Comment (Flova) [reply msg/RobotRelative.msg:5]: Todo check the comment from @phinik in this context

**Pull request merged**